lighthouse-core/scripts/save-latest-run.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+# Saves the necessary contents of the `latest-run/` folder to a subfolder for easier A/B comparison.
+# Restoring the contents to `latest-run/` is just `cp latest-run/latest-run.bak/* latest-run/`.
+
+DIRNAME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+LH_ROOT="$DIRNAME/../.."
+TARGET_DIR=${1:-latest-run.bak}
+cd $LH_ROOT/latest-run
+
+mkdir -p $TARGET_DIR
+cp artifacts.json $TARGET_DIR/
```

Comment:
missing lhr..want to just copy every file?

Reply:
> missing lhr.

that was somewhat intentional but I suppose it wouldn't hurt :)

> want to just copy every file?

not really since I keep lots of other stuff in there 😆 , but now that you mention it I think we need to for runtimeError artifacts anyway. I'll move my stuff and copy all files 👍